zenith_metrics/src/lib.rs

```diff
@@ -74,6 +74,10 @@ lazy_static! {
     .expect("Failed to register maxrss_kb int gauge");
 }
 
+pub const DISK_WRITE_SECONDS_BUCKETS: &[f64] = &[
+    0.000_050, 0.000_100, 0.000_500, 0.001, 0.003, 0.005, 0.01, 0.05, 0.1, 0.3, 0.5,
```

Comment:
* These numbers are fairly arbitrary, suggestions on better buckets are welcome.
* Note that these buckets are currently shared between two histograms: WAL and control file write+sync latency. They may be shared further in the future. I assumed that we would (almost?) never spend more than 0.5s writing to disk, not sure if this true.